client/src/homepage.css

```diff
@@ -0,0 +1,55 @@
+*,
+*::after*::before {
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+
+	font-family: 'Montserrat', sans-serif;
+}
```

Comment:
In general, we should use something like **reset.css** that is imported in main tsx file (App.tsx) and it should contain this selector.